src/components/MobileNavigation/MobileNavigation.scss

```diff
@@ -0,0 +1,42 @@
+.MobileNavigation__wrapper {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 32px;
+    gap: 50px;
+    height: 798px;
+    left: 0px;
+    top: 96px;
+    background: var(--color-white-lilac);
+    border-bottom: 1px solid #e3e0ed;
+
+    &__link1, &__link2 {
+        font-style: normal;
+        font-weight: 600;
+        font-size: 14px;
+        line-height: 17px;
+        color: var(--color-bunting);
+        flex: none;
+        order: 0;
+        flex-grow: 0;
```

Comment:
you don't need any flex/order/grow here, but could you please add  text-decoration: none; to remove the underline